medips.R

```diff
@@ -29,20 +31,14 @@ uniq <- opt$uniq
 extend <- opt$extend
 shift <- opt$shift
 ws <- opt$ws
+chromosome <- opt$chromosome
 samplename <- opt$samplename
-simpleCigarFlag <- opt$cigarFlag
-covX <- opt$covX
-
-library(MEDIPS)
-library(BSgenome.Hsapiens.UCSC.hg19)
-library(BSgenome.Hsapiens.UCSC.hg38)
-library(MEDIPSData)
 
 # set the working dir
 setwd(basedir)
 
 # set chromosomes
-chr.select=paste0("chr",c(1:22,"X","Y"))
+chr.select=ifelse(is.null(chromosome), paste0("chr",c(1:22,"X","Y")), chromosome)
```

Comment:
This is where we set a chromosome. The behaviour of the previous version is preserved. It will proceed as before when no chromosome argument is passed

Reply:
makes sense.  this is so we can pass a single chromosome

Reply:
yes, this is what enables us to use scatter in the workflow